index.html

```diff
@@ -87,6 +91,8 @@
             <br>
         </div>
         <br>
+        <div class="plyr"><audio></audio></div>
```

Comment:
Should I revert? Reset? Plyr loads and is able to play with `<div>` but it stops somewhere with:

> Uncaught TypeError: Cannot read property 'play' of undefined

and because it stops there it doesn't even load controls' icons.